catalog.html

```diff
@@ -24,154 +23,172 @@ <h1>
      </h1>
     </div>
     <div>
-     +7 (812) 555-05-55<br>
-     г.Санкт-Петербург, ул. Б. Конюшенная, д. 19/8
+      <a href="tel:+78125550555">+7 (812) 555-05-55</a><br>
+      <address>г.Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</address>
     </div><br>
+    <a href="#"><img src="img/icon-user.svg" alt="Икнока пользователя для входа"></a>
     <button>Равшан Джамшутович</button>
+    <a href="#"><img src="img/icon-login.svg" alt="Иконка выхода из личного кабинета"></a>
     <ul>
      <li><a href="#">Мои заказы</a>
      <li><a href="#">Личный кабинет</a>
     </ul>
     <ul>
      <li><a href="index.html">Главная</a></li>
      <li><a href="#">Компания</a></li> <!--ссылка на раздел о Компании-->
-     <li><a href="#">Каталог</a></li>  <!--ссылка в Каталог-->
+     <li><a>Каталог</a></li>  <!--ссылка в Каталог-->
      <li><a href="#">Новости</a></li>  <!--ссылка на раздел Новости-->
      <li><a href="#">Спецпредложения</a></li> <!--ссылка на раздел Спецпредложения-->
      <li><a href="#">Доставка</a></li> <!--ссылка на раздел Доставка-->
      <li><a href="#">Контакты</a></li> <!--ссылка на раздел Контакты-->
     </ul>
    </header>
    <main>
+     <ul>
+       <li><a href="index.html"><img src="img/icon-home.svg" alt="Иконка-ссылка для перехода на главную"></a></li>
+       <li><a href="catalog.html">Каталог</a></li>
+       <li><a href="#">Инструмент</a></li>
+       <li><a>Перфораторы</a></li>
+     </ul>
      <h1>Перфораторы</h1>
```

Comment:
```suggestion
     <h1>Перфораторы</h1>
     <ul>
       <li><a href="index.html"><span class="visually-hidden">Главная"></span></a></li>
       <li><a href="catalog.html">Каталог</a></li>
       <li><a href="#">Инструмент</a></li>
       <li><a>Перфораторы</a></li>
     </ul>
```
Заголовок и визуально (в макете) расположен первым и логически (в разметке) должен идти первым. Иконка - декоративный элемент.